Corcav.Behaviors/Library/EventToCommand.cs

```diff
@@ -148,7 +148,7 @@ private void AddEventHandler(EventInfo eventInfo, object item, Action<EventArgs>
         /// <param name="e">The EventArgs value accompanying the Event</param>
         private void OnFired(EventArgs e)
 		{
-            object returnParam = this.CommandParameter ?? e;
+            object returnParam = e ?? this.CommandParameter;
```

Comment:
Hello
In my opinion this line introduces a new bug. And it should be changed back. It is now impossible to pass in a CommandParameter if the event sends Arguments. 
For example the following code always passes EventArgs instead of the bound value. 

` <b:EventToCommand CommandNameContext="{b:RelativeContext FolderPage}"
                                        EventName="Clicked" CommandName="InfoClickedCommand"
                                        CommandParameter="{Binding .}" />`